backend/spec/features/admin/orders/new_order_spec.rb

```diff
@@ -130,7 +130,7 @@
       click_on 'Cart'
       select2_search product.name, from: Spree.t(:name_or_sku)
 
-      fill_in_quantity('table.stock-levels', 'quantity_0', 1)
```

Comment:
I think we can also deprecate and remove the `fill_in_quantity` Capybara extension as well.